config/Config-kernel.in

```diff
@@ -1204,6 +1204,18 @@ config KERNEL_PAGE_POOL_STATS
 	bool "Page pool stats support"
 	depends on KERNEL_PAGE_POOL
 
+config KERNEL_MPTCP
+	bool "Multi-Path TCP support"
+	default y if !SMALL_FLASH
+	help
+	   Select this option to enable support for Multi-Path TCP.
+	   Increases the compressed kernel size by ~214kB (as of Linux 6.6).
+
+config KERNEL_MPTCP_IPV6
+	bool "IPv6 support for Multipath TCP"
+	depends on KERNEL_MPTCP
```

Comment:
Please make this option depend on `KERNEL_IPV6` or move it to the block inside the `if KERNEL_IPV6` block.